packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -124,7 +124,8 @@ export const getAvailableContractTypes = (contract_types_list: TContractTypesLis
 //     );
 
 export const findContractCategory = (list: Partial<TList[]>, item: TItem) =>
-    list?.find(list_item => list_item?.contract_types?.some(i => i.value === item.value)) || ({} as TContractCategory);
+    list?.find(list_item => list_item?.contract_types?.some(i => i.value.includes(item.value))) ||
```

Comment:
@akmal-deriv will it affect `rise_fall` and `rise_fall_equal` ?

Reply:
This particular change is here to better handle contracts which would have the same trade type, e.g vanilla, but different content/UI for specific symbols, e.g. vanilla_fx. It shouldn't break any existing trade type if the content/UI are the same.